### ISSUE TITLE
Signet faucet?

### ISSUE THREAD

**Issue by Purchase73** [thunderbiscuit/padawan-wallet#394]
Hi! I'm looking to put together some bitcoin educational material, and was planning on using Signet (after reading about Testnet being more difficult to use).

However, all the signet faucets I can find don't seem to be working. As a matter of fact, the only place I was able to get signet coins was via your app (thank you!).

Do you have any thoughts on how to reliably have students independently acquire signet?

**Comment by thunderbiscuit** [comment]
No unfortunately I have also had a hard time getting access to signet coins! The few faucets that were live are not working anymore, and in the end I had to have someone sending me some directly. Testnet 4 might be the answer here once it gets more integrated into people's workflows? Still too young for me and my use case for the app, but I'm keeping an eye on it.

Good luck, and thanks for checking out Padawan!

**Comment by Purchase73** [comment]
Thanks so much for your reply!

I don't know a lot about the different options (signet, testnet4, etc), just what I've found in an online search.

What do you think the negatives are for testnet4? I'm mainly looking to write up some practice exercises for Sparrow 2.0. With the limited tests I've done, it looks like it Sparrow works with testnet4.

I'd love to just point people to the faucets, but it seems like they can be unreliable and stop working at any time.

thanks!

Sent with [Proton Mail](https://proton.me/mail/home) secure email.

**Comment by thunderbiscuit** [comment]
Yeah faucets are super unreliable in general. Take a look at [this issue](https://github.com/thunderbiscuit/padawan-wallet/issues/376) for my thoughts on Testnet4 at the moment.

But for what you want to do, I think it might work well! Notice that there is a list of 3 supposedly live faucets for testnet 4 at the moment: https://testnet4.com/.

**Comment by ajtowns** [comment]
For devs, probably the easiest solution is to join the signet dev discord, which has a faucet bot that's working. https://x.com/ajtowns/status/1862163340836901255 I'll see if I can make that bot also work for people who have a github account via posting to nostr.

**Comment by thunderbiscuit** [comment]
Awesome suggestion @ajtowns! I joined a few weeks back. Great stuff, thanks for maintaining it.

**Comment by ajtowns** [comment]
These days https://signet25.bublina.eu.org/ seems to be reasonably reliable
